File: DTOs/Sources/MeetingServiceDTOs/CreateVotingDTO.swift
import Foundation

public struct CreateVotingDTO: Codable {
    public var meetingId: UUID
    public var question: String
    public var description: String?
    public var anonymous: Bool
    public var options: [GetVotingOptionDTO]
    
    public init(meetingId: UUID, question: String, description: String? = nil, anonymous: Bool, options: [GetVotingOptionDTO]) {
        self.meetingId = meetingId
        self.question = question
        self.description = description
        self.anonymous = anonymous
        self.options = options
    }
}

File: android/app/src/main/java/net/ipv64/kivop/dtos/MeetingServiceDTOs/CreateVotingDTO.kt
package net.ipv64.kivop.dtos.MeetingServiceDTOs

import java.util.UUID

data class CreateVotingDTO(
    var meetingId: UUID,
    var question: String,
    var description: String?,
    var anonymous: Boolean,
    var options: List<GetVotingOptionDTO>,
)
